src/ContainerTerminal.jsx

```diff
@@ -259,10 +259,12 @@ class ContainerTerminal extends React.Component {
         if (this.props.containerStatus !== "running" && !this.state.opened)
             element = <EmptyStatePanel title={_("Container is not running")} />;
 
-        return <>
-            {this.state.errorMessage && <ErrorNotification errorMessage={_("Error occurred while connecting console")} errorDetail={this.state.errorMessage} onDismiss={() => this.setState({ errorMessage: "" })} />}
-            {element}
-        </>;
+        return (
+            <>
+                {this.state.errorMessage && <ErrorNotification errorMessage={_("Error occurred while connecting console")} errorDetail={this.state.errorMessage} onDismiss={() => this.setState({ errorMessage: "" })} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1217-20230227-180700-70c39418-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)